### ISSUE TITLE
Legendary items ?

### ISSUE THREAD

**Issue by ObaniGemini** [cdqwertz/rpgtest#1]
" Do you have any ideas for legendary items? "

Let's post ideas here !

- Dragon stuff (bows, swords, armors)
- Phenix stuff (bows, magical items/sticks)
- Legendary lockets (magical powers, like thunder/seism/whatever you want)
- ...etc...

In general : 
- Objects made with or by legendary beasts/tribes (e.g : elves)
- Objects giving great powers to the user, like permitting to teleport, fly, go fast, being stronger, jumping higher,...
- They might be possible to find in chests or by killing some kinds of monsters (but with high rarity factor)

**Comment by cdqwertz** [comment]
maybe you saw, that I already have done some legendary items :)
But, I think about 5 legendary items are not enough :)
@ObaniGemini : Its a great idea to open an Issue for this! Thank you :)

**Comment by farfadet46** [comment]
To get "super powers" i think potion is cool for a little time.
Legendary is for timeless :)

**Comment by farfadet46** [comment]
when you say Dragon, Phoenix,... you say there is a pet (or boss) with this name ?

**Comment by ObaniGemini** [comment]
no. And I didn't except the idea of a boss or of a dungeon. But you usually find chests in a dungeon and bosses are hard to make well.
